organizations/admin.py

```diff
@@ -1,65 +1,63 @@
-# pylint: disable=too-many-public-methods
-# pylint: disable=no-member
-"""
-django admin pages for organization models
-"""
+""" Django admin pages for organization models """
 from django.contrib import admin
+from django.utils.translation import ugettext_lazy as _
+
 from organizations.models import (Organization, OrganizationCourse)
 
 
+@admin.register(Organization)
 class OrganizationAdmin(admin.ModelAdmin):
-    """
-    Admin for the Organization table.
-    soft-delete on the organizations
-    """
-    list_display = ('name', 'short_name', 'description', 'logo', 'created', 'active')
-    readonly_fields = ('created',)
-    ordering = ['created']
+    """ Admin for the Organization model. """
     actions = ['activate_selected', 'deactivate_selected']
+    list_display = ('name', 'short_name', 'logo', 'active',)
+    list_filter = ('active',)
+    ordering = ('name', 'short_name',)
+    readonly_fields = ('created',)
+    search_fields = ('name', 'short_name',)
 
     def get_actions(self, request):
-        """ return actions """
         actions = super(OrganizationAdmin, self).get_actions(request)
+
+        # Remove the delete action.
         del actions['delete_selected']
+
         return actions
 
     def activate_selected(self, request, queryset):
-        """activate the selected entries"""
+        """ Activate the selected entries. """
         queryset.update(active=True)
+
         if queryset.count() == 1:
-            message_bit = "1 organization entry was"
+            message = _('1 organization entry was successfully activated')
         else:
-            message_bit = "%s organization entries were" % queryset.count()
-        self.message_user(request, "%s successfully activated." % message_bit)
+            message = _('%s organization entries were successfully activated') % queryset.count()
```

Comment:
Django caches that information. Nevertheless, I've moved it to a variable to avoid confusion in the future.